### PR TITLE
Bring KMS client and multiclient over to chainlink

### PR DESCRIPTION
**Pull request by ogtownsend**

### CODE REVIEW

**Review by AnieeG — COMMENTED** [review]
You will need to run go mod tidy in integration-tests

**Review by AnieeG — DISMISSED** [review]

**Review by connorwstein — DISMISSED** [review]

File: core/capabilities/ccip/ccip_integration_tests/helpers.go [hunk 1]
```diff
@@ -406,7 +406,6 @@ func (h *homeChain) AddNodes(
 	p2pIDs [][32]byte,
 	capabilityIDs [][32]byte,
 ) {
-	// Need to sort, otherwise _checkIsValidUniqueSubset onChain will fail
 	sortP2PIDS(p2pIDs)
```
Comment (makramkd) [inline core/capabilities/ccip/ccip_integration_tests/helpers.go:409]: Pretty sure this and the associated function def'n can be deleted, sorting is no longer a requirement

Comment (makramkd) [reply core/capabilities/ccip/ccip_integration_tests/helpers.go:409]: Anyway, not required to do in this PR ;-)

Comment (connorwstein) [reply core/capabilities/ccip/ccip_integration_tests/helpers.go:409]: I think this whole directory could probably deleted no? Thats what I had in mind with CCIP-3049

Comment (makramkd) [reply core/capabilities/ccip/ccip_integration_tests/helpers.go:409]: If these helpers aren't used anywhere and these tests are not needed, yes.

**Review by AnieeG — APPROVED** [review]

**Review by connorwstein — APPROVED** [review]

**Pull request merged**